web/src/lottie-player.ts

```diff
@@ -360,7 +360,7 @@ export class LottiePlayer extends LitElement {
     }
 
     const buffer = this._TVG.render();
-    const clampedBuffer = Uint8ClampedArray.from(buffer);
+    const clampedBuffer = new Uint8ClampedArray(buffer.buffer, buffer.byteOffset, buffer.byteLength);
```

Comment:
@theashraf could you please review this in dotlottie side? This might help dotlottie player to improve the fps (~5fps)

Reply:
Awesome @hermet

Reply:
@hermet in dotlottie-web, we're currently not casting the framebuffer, which should bring better performance since the ThorVG framebuffer is guaranteed to be a Uint8 Array anyway.

https://github.com/LottieFiles/dotlottie-web/blob/991635ac4fefedfe98cae9f3dd69d66273323d89/packages/web/src/dotlottie.ts#L457

Reply:
@theashraf thanks, we will double check it. 👍